Orange/widgets/data/owdatasets.py

```diff
@@ -171,11 +172,17 @@ class Outputs:
 
     def __init__(self):
         super().__init__()
+        self.allinfo_local = {}
+        self.allinfo_remote = {}
+
         self.local_cache_path = os.path.join(data_dir(), self.DATASET_DIR)
 
-        self._header_labels = [header['label'] for _, header in self.HEADER_SCHEMA]
-        self._header_index = namedtuple('_header_index', [info_tag for info_tag, _ in self.HEADER_SCHEMA])
-        self.Header = self._header_index(*[index for index, _ in enumerate(self._header_labels)])
+        self._header_labels = [
+            header['label'] for _, header in self.HEADER_SCHEMA]
+        self._header_index = namedtuple(
+            '_header_index', [info_tag for info_tag, _ in self.HEADER_SCHEMA])
+        self.header = self._header_index(
```

Comment:
Changing `Header` to `header` breaks the scOrange datasets widget, which extends this one and redefines `Header`.

I see no reason why it was capitalized before, so I agree with the change. But it needs to be coordinated with changes in widgets that extend this one.